frontend/javascripts/oxalis/model/reducers/volumetracing_reducer.ts

```diff
@@ -194,6 +198,37 @@ function handleUpdateSegment(state: OxalisState, action: UpdateSegmentAction) {
   });
 }
 
+function handleFocusSegment(state: OxalisState, action: ClickSegmentAction) {
+  if (action.layerName == null) return state;
+  const getNewGroups = () => {
+    const { segments, segmentGroups } = getVisibleSegments(state);
+    if (action.layerName == null || segments == null) return segmentGroups;
+    const { segmentId } = action;
+    const segmentForId = segments.getNullable(segmentId);
+    if (segmentForId == null) return segmentGroups;
+    // Expand recursive parents of group too, if necessary
+    let pathToRoot = [];
```

Comment:
wouldn't `const` work, too? also, you can initialize this as a Set, since you will need a set later, anyway. instead of `push` you only need to call `add`.